deploy.sh

```
@@ -0,0 +1,179 @@
+# Ensure Docker starts on boot and start Docker service
+sudo systemctl enable docker
+sudo systemctl start docker
+
+# Clone the Git repository
+if [ -d "$APP_DIR" ]; then
+  echo "Directory $APP_DIR already exists. Pulling latest changes..."
+  cd $APP_DIR && git pull
+else
+  echo "Cloning repository from $REPO_URL..."
+  git clone $REPO_URL $APP_DIR
+  cd $APP_DIR
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)